newsblur_web/docker_local_settings.py

```diff
@@ -170,8 +170,8 @@
 SERVER_NAME = "nblocalhost"
 NEWSBLUR_URL = os.getenv("NEWSBLUR_URL", "https://localhost")
 
-if NEWSBLUR_URL == 'https://nb.local.com':
-    SESSION_COOKIE_DOMAIN = ".nb.local.com"
+if NEWSBLUR_URL == 'https://localhost':
+    SESSION_COOKIE_DOMAIN = "localhost"
```

Comment:
Ahh this is why nb.local.com was removed above. I think we can keep it in while preserving this change.